qiskit/transpiler/passes/basis/basis_translator.py

```diff
@@ -486,8 +543,7 @@ def _compose_transforms(basis_transforms, source_basis, source_dag):
             source_basis but not affected by basis_transforms will be included
             as a key mapping to itself.
     """
-
-    example_gates = {(node.op.name, node.op.num_qubits): node.op for node in source_dag.op_nodes()}
+    example_gates = _get_example_gates(source_dag, defaultdict(set))
```

Comment:
Is the `defaultdict(set)` necessary here?  The output hashmap seems to be of type `(str, int): Instruction`, not `set`, given the definition of `_get_example_gates`.

Reply:
I can't remember why this was here. Perhaps for the linter? Using your suggestion for get_example_gates changed the signature.